src/main/java/io/meeds/qa/ui/pages/page/factory/settings/SettingsPage.java

```diff
@@ -10,25 +10,25 @@
 
 public class SettingsPage extends GenericPage {
   @FindBy(
-      xpath = "(//button[@class='btn btn-primary v-btn v-btn--contained theme--light v-size--default']//*[@class='v-btn__content'])[1]"
+          xpath = "(//button[@class='btn btn-primary v-btn v-btn--contained theme--light v-size--default']//*[@class='v-btn__content'])[1]"
```

Comment:
please format only the changed code and not the whole file, you select the introduced change, and then format it